adafruit_ht16k33/segments.py

```diff
@@ -289,12 +289,13 @@ def _number(self, number: Union[int, float], decimal: int = 0) -> str:
                 places += 1
 
         # Set decimal places, if number of decimal places is specified (decimal > 0)
+        txt = None
```

Comment:
Should this maybe be `txt = stnum`? I'm thinking about numbers without decimal places in them.

Reply:
Oh, probably.  The original intention was to not changed exiting behavior, just to guarantee that `txt` was never unbound.  But I think you're right, `txt` probably should be set to `stnum`.  I can update that.